styles.css

```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap');
+@import url('https://elegantui.netlify.app/styles.css');
+
+html{
+    font-size: 100%;
+    font-family: "Noto Sans", sans-serif;
+    scroll-behavior: smooth;
+}
+
+:root{
+    --color-primary:#991b1b;
+    --color-secondary:antiquewhite;
+}
+
+.wrapper-landing{
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    height: 100%;
+}
+
+.section-header{
+    width:100%;
+    min-height: 6rem;
+    max-height:10%;
+    background-color: var(--color-secondary);
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    flex-wrap: wrap;
+    gap:1rem;
+}
+
+.logo-title{
+    max-width:15%;
+    margin-left: 1rem;
+    font-size: 1.5rem;
+    cursor: pointer;
+}
+
+.search-product{
+    flex-grow:1;
+    justify-content: space-around;
+}
+
+.search-bar{
+    display:flex;
+    flex-wrap: wrap;
+    width:70%;
+    margin: auto;
+}
+
+.search-bar input{
+    flex-grow: 1;
+}
+
+.search-bar input[type=text] {
+    padding: 0.75rem;
+    font-size: 1.1rem;
+    border: 0.2rem solid var(--color-primary);
+    background: var(--color-white);
+    border-radius: 5px;
+}
+  
+.section-header button,
+.primary-btn-link{
+    background-color: var(--color-secondary);
+    color:var(--color-primary);
+    padding:0.1rem 0.1rem;
+    margin:0.2rem 0.2rem;
+    width:auto;
```

Comment:
Width by default is auto, and If the value is the same for the top,bottom and left,right you can use one value,

Reply:
changes done